pep/app/pods/index/controller.ts

```diff
@@ -20,7 +20,7 @@ export default class Index extends Controller {
     }
 
     get expertPick() {
-        return this.configuration.base.home.expertPick;
+        return this.configuration.base.home.expertPicks[0];
```

Comment:
this should _probably_ be `expertPicks[expertPicks.length - 1]` to show the last one in the list, as i believe they want to show the most recently added one here, but i could be wrong on that.